microservices-connector/cmd/router/main.go

```diff
@@ -198,6 +199,40 @@ func executeStep(
 	return callService(step, serviceURL, input, headers)
 }
 
+func mergeRequests(initReq, respReq []byte) []byte {
+	var initReqData map[string]interface{}
+	var respReqData map[string]interface{}
+
+	if err := json.Unmarshal(initReq, &initReqData); err != nil {
```

Comment:
please do not unmarshal intiReq every time to save time.

Reply:
Hi @irisdingbj, okay, it makes sense, done!